extensions/healthie/actions/getMostRecentMetricEntry/getMostRecentMetricEntry.ts

```diff
@@ -0,0 +1,87 @@
+import { ZodError, z } from 'zod'
+import { fromZodError } from 'zod-validation-error'
+import { validate, type Action } from '@awell-health/extensions-core'
+import { Category } from '@awell-health/extensions-core'
+import { type settings, settingsValidationSchema } from '../../settings'
+import { HealthieError, mapHealthieToActivityError } from '../../errors'
+import { dataPoints, fields, fieldsValidationSchema } from './config'
+import { initialiseClient } from '../../graphqlClient'
+import { getSdk } from '../../gql/sdk'
+
+export const getMostRecentMetricEntry: Action<
+  typeof fields,
+  typeof settings,
+  keyof typeof dataPoints
+> = {
+  key: 'getMostRecentMetricEntry',
+  category: Category.EHR_INTEGRATIONS,
+  title: 'Get most recent metric entry',
```

Comment:
After reviewing, I would prefer renaming the action (and the files) to "Get metric entry". Can you do that in all places?

```suggestion
  title: 'Get metric entry',
```